supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -45,6 +45,17 @@
 
 *See also*:
 
+[discrete]
+[[ACK]]
+==== image:images/yes.png[yes] ACK flag (noun)
+*Description*: The acknowledgement flag ("ACK flag") in a TCP packet indicates that the acknowledgement field in the packet is relevant. Do not use "ACK" as acronym for "acknowledgement" in general.
```

Comment:
@mmuehlfeldRH We're in the middle of trying to update the SSG to be consistent with how we use italics and quotes. Can you do a sweep through these entries and try to make updates so that it matches the approach we've identified here: https://github.com/redhat-documentation/supplementary-style-guide/issues/173#issuecomment-1118584024

Basically if we're defining the term at the beginning, use italics for it. But then when we're saying what to write or not write, use quotes. For example, this entry I would change like this:

```suggestion
*Description*: The acknowledgement flag, also known as the _ACK flag_, in a TCP packet indicates that the acknowledgement field in the packet is relevant. Do not use "ACK" as acronym for "acknowledgement" in general.
```

Not sure if the "also known as" part is correct or not, or if you were trying to say that it should be written out as "acknowledgment flag (ACK flag)" upon first mention.

Reply:
I updated all entries with respect to quotes and italic, and I made clear in "ACK flag" how to write it.